docs/schedules.md

```diff
@@ -0,0 +1,145 @@
+Scheduled Jobs
+==============
+
+awx allows jobs to run on a schedule (with optional recurrence rules) via
+an `HTTP POST` to a variety of API endpoints:
+
+    HTTP POST
+
+    https://tower-host.example.org/api/v2/job_templates/N/schedules/
+    https://tower-host.example.org/api/v2/projects/N/schedules/
+    https://tower-host.example.org/api/v2/inventory_sources/N/schedules/
+    https://tower-host.example.org/api/v2/system_jobs/N/schedules/
+    https://tower-host.example.org/api/v2/workflow_job_templates/N/schedules/
+
+    {
+        'name': 'My Schedule Name',
+        'rrule': 'DTSTART:20300115T120000Z RRULE:FREQ=DAILY;INTERVAL=1;COUNT=7'
+        'extra_data': {}
+    }
+
+...where `rrule` is a valid
+[RFC5545](https://www.rfc-editor.org/rfc/rfc5545.txt) RRULE string.  The
+specific example above would run a job every day - for seven consecutive days - starting
+on January 15th, 2030 at noon (UTC).
+
+Specifying Timezones
+====================
+`DTSTART` values provided to awx _must_ provide timezone information (they may
+not be naive dates).
+
+For UTC dates, `DTSTART` values should be denoted with the `Z` suffix:
+
+    DTSTART:20300115T120000Z
+
+Local timezones can be specified using the `TZID=` parameter:
+
+    DTSTART;TZID=America/New_York:20300115T120000Z
```

Comment:
`Z` should be omitted here since we're dealing with a non-UTC timezone, correct?

Reply:
Yep, good catch.